src/Abp.Zero/Authorization/Roles/AbpRoleManager.cs

```diff
@@ -400,7 +400,8 @@ private Task<TRole> FindByDisplayNameAsync(string displayName)
         private async Task<RolePermissionCacheItem> GetRolePermissionCacheItemAsync(int roleId)
         {
             var cacheKey = roleId + "@" + (GetCurrentTenantId() ?? 0);
-            return await CacheManager.GetRolePermissionCache().GetAsync(cacheKey, async () =>
+
+            var cacheItem = await CacheManager.GetRolePermissionCache().GetAsync(cacheKey, async () =>
```

Comment:
is it possible to remove `var cacheItem =` and just use the return ?

Reply:
Yes, it was before but I changed it to debug friendly. If you think better to use just return, I can change?

Reply:
it is better to use return I think.

Reply:
Done!